src/org/usfirst/frc/team2399/robot/subsystems/Agitator.java

```diff
@@ -0,0 +1,35 @@
+package org.usfirst.frc.team2399.robot.subsystems;
+
+import org.usfirst.frc.team2399.robot.RobotMap;
+import org.usfirst.frc.team2399.robot.commands.AgitatorStop;
+
+import com.ctre.CANTalon;
+import com.ctre.CANTalon.TalonControlMode;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class Agitator extends Subsystem {
+
+	private CANTalon agitatorTalon;
+	
+	public Agitator(){
+		agitatorTalon = new CANTalon(RobotMap.AGITATOR_TALON_ADDRESS);
+		agitatorTalon.changeControlMode(TalonControlMode.Speed);
```

Comment:
Are you sure this is doing what you think it does? What units does the talon expect in set() in speed mode?